cmd/insight.go

```diff
@@ -26,8 +26,8 @@ var insightCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(insightCmd)
 
-	insightCmd.Flags().StringVar(&insightOptions.GRPC, "gRPC", "", "gRPC server information")
-	insightCmd.Flags().StringVar(&insightOptions.Source, "source", "all", "The DB for insight : system|network|all")
+	insightCmd.Flags().StringVar(&insightOptions.GRPC, "grpc", "", "gRPC server information")
+	insightCmd.Flags().StringVar(&insightOptions.Source, "class", "all", "The DB for insight : system|network|all")
```

Comment:
Also change the names here in the help command "The DB for insight : application|network|all", you will also need to modify the logic in the insight/insight.go file

Reply:
🤔  @Ankurk99 not sure what part of insight.go need to be modified. Right now I'm guessing somewhere here
https://github.com/kubearmor/kubearmor-client/blob/f645bd1a01257bf6424642a203a2da1fe23c7958/insight/insight.go#L27 ?

Reply:
kubearmor-client insights are [fetched](https://github.com/kubearmor/kubearmor-client/blob/main/insight/insight.go#L66) from the [discovery-engine](https://github.com/accuknox/discovery-engine). If you check in discovery-engine, we are [checking for network/system insights](https://github.com/accuknox/discovery-engine/blob/dev/src/insight/insight.go#L75). You will have to create a condition in insight/insight.go to match and check for "network/application" condition instead.

Reply:
Oh, Thanks you for clearing that up

Reply:
one more thing @Ankurk99  shouldn't this be changed to "Class" https://github.com/kubearmor/kubearmor-client/blob/f645bd1a01257bf6424642a203a2da1fe23c7958/insight/insight.go#L25 ?

Reply:
Yes, also please change it in cmd files too.